src/models/Tickets.ts

```diff
@@ -1,17 +1,16 @@
-import { Schema, model } from 'mongoose';
+import mongoose, { Schema, model } from 'mongoose';
 
-const ticketSchema = new Schema({
+const TicketSchema = new Schema({
+  id: mongoose.Types.ObjectId,
   subject: String,
   status: String,
-  users: Array,
   deadline: Date,
   description: String,
   initial_time_estimated: Number,
   total_time_spent: Number,
   advancement: Number,
-  file_links: Array
```

Comment:
`file_links` is not in `Ticket` model anymore?

Reply:
Not for now?